geyser-plugin-interface/src/geyser_plugin_interface.rs

```diff
@@ -13,6 +15,37 @@ use {
     thiserror::Error,
 };
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+#[repr(C)]
+/// Information about a node  in  the cluster.
+pub struct ReplicaClusterInfoNode {
```

Comment:
What happens to this struct if we change gossip `ContactInfo`, e.g. add a new socket for a new protocol?

Reply:
ya so that is the goal of doing something with an FFI interface. See PR: [#3434](https://github.com/anza-xyz/agave/pull/3434). We could modify that PR and choose to only expose a pointer to a ContactInfo struct, making it much easier to update `ContactInfo` in the future. More details here: https://github.com/anza-xyz/agave/pull/3434#issuecomment-2469158222